src/SingleCellAnnotator.imjoy.html

```diff
@@ -147,14 +147,26 @@
     if(!color_channels[j].length){
       continue
     }
-    for(let k=0; k<color_channels[j].length; k++){
-      src = color_channels[j][k].data;
-      for(i=0;i<channel_offset;i++){
-          pixel_index = i*4+j
-          v = data[pixel_index];
-          v = v + src[i]
-          if(brightness!==1) v = v*brightness;
-          data[pixel_index] = v>255?255:parseInt(v);
+    if(brightness===1){
+      for(let k=0; k<color_channels[j].length; k++){
+        src = color_channels[j][k].data;
+        for(i=0;i<channel_offset;i++){
+            pixel_index = i*4+j
+            v = data[pixel_index];
+            v = v + src[i]
+            data[pixel_index] = v>255?255:v;
```

Comment:
replace parseint(v) with v here, since if without brightness adjustment, v is int